app/frontend/styles/_table.scss

```diff
@@ -0,0 +1,33 @@
+.app-table__cell--action {
+  text-align: right;
+}
+
+// This cell contains a prompt for addressing a break in work history.
+.app-table__cell--break-prompt {
+  border-left: $govuk-border-width solid govuk-colour("blue");
+  @include govuk-responsive-padding(4, "left");
+  background-color: govuk-colour("light-grey");
+}
+
+
+// Modifiers that make captions look more like their equivalent headings
+// This is being added to govuk-frontend soon:
+// https://github.com/alphagov/govuk-frontend/pull/2048
+.app-table__caption--xl {
+  @include govuk-font($size: 48, $weight: bold);
+  margin-bottom: govuk-spacing(3);
+}
+
+.app-table__caption--l {
+  @include govuk-font($size: 36, $weight: bold);
+  margin-bottom: govuk-spacing(3);
+}
+
+.app-table__caption--m {
+  @include govuk-font($size: 24, $weight: bold);
+  margin-bottom: govuk-spacing(3);
+}
+
+.app-table__caption--s {
+  @include govuk-font($size: 19, $weight: bold);
+}
```

Comment:
These can be dropped now, as they've been added upstream.

```suggestion
```

Reply:
👍